### DISCUSSION TITLE
getting_started/

### DISCUSSION THREAD

**Discussion by giscus** [Air-duino/document#15] [comment]
# getting_started/

AirMCU 是一个由社区推动，旨在适配上海合宙通信有限公司的 Arm-Cortex 系列 MCU 的 Arduino 框架。 支持芯片 目前我们支持的 MCU 如下： MCU 系列 是否支持 详细链接 :-------: :------: :-----------------: Air001 ✅ Air32F103 🔨 有关开发板的更多资料，请查看...

https://arduino.luatos.com/getting_started/

**Comment by Zhentao-Lin** [comment]
示例中，没有eeprom示例吗

**Reply by chenxuuu** [reply]
包里就有例子

**Reply by Zhentao-Lin** [reply]
方便告知在示例中的名称吗

**Comment by xxkj2010** [comment]
Arduino 1.8.19无法安装air001开发板

**Reply by xxkj2010** [reply]
搜索不到该开发板

**Reply by Zhentao-Lin** [reply]
只支持2.0.0以上版本下载运行板级支持包。1.8.x版本用不了的

**Reply by xxkj2010** [reply]
没有自动下载功能，Arduino 要下载程序很难控制得好，基本上都超时失败。

**Reply by chenxuuu** [reply]
有自动下载功能，文档里的烧录方法写的很详细

**Comment by xxkj2010** [comment]
请问air001是否需要先写入bootloader才能进行Arduino串口下载？
